build-scripts/update_sds_to_scap_1_3.py

```diff
@@ -0,0 +1,139 @@
+
+from __future__ import print_function
+
+import sys
+try:
+    from urllib.parse import urlparse
+except ImportError:
+    from urlparse import urlparse
+
+import ssg.constants
+import ssg.xml
+ocil_ns = ssg.constants.ocil_namespace
+
+oval_ns = ssg.constants.oval_namespace
+xccdf_ns = ssg.constants.XCCDF12_NS
+ds_ns = ssg.constants.datastream_namespace
+xlink_ns = ssg.constants.xlink_namespace
```

Comment:
This is just renaming variables. Please use them directly.

Reply:
I have updated the import definition and changed code to use them.